libc/src/ckb_cell_fs.c

```diff
@@ -17,9 +17,20 @@ int get_file(const CellFileSystem *fs, const char *filename, FSFile **f) {
     CellFileSystem *cfs = (CellFileSystem *)fs;
     CellFileSystemNode *node = cfs->current;
     while (node != NULL) {
+        size_t filename_len = strlen(filename);
+        size_t prefix_len = strlen(node->prefix) - 1;
```

Comment:
`node->prefix` should be compared to filename at the beginning.

Reply:
Fixed